client/src/components/ElementHistoryViewer/ElementHistoryViewer.js

```diff
@@ -0,0 +1,14 @@
+// import React from 'react';
+import { compose } from 'redux';
+import readOneBlockQuery from 'state/history/readOneBlockQuery';
+
+/**
+ * Transformer function which binds the readOneBlockQuery GraphQL query to the HistoryViewer
+ * component.
+ */
+const withElementsHistoryViewer = (HistoryViewer) => compose(
+  readOneBlockQuery
+)(HistoryViewer);
+
+export default withElementsHistoryViewer;
```

Comment:
We've removed this file deliberately, since it's handled purely in the registerTransforms file now. Is there a reason it was added back in?